tests/integration/SingleEndVariantCalling.json

```diff
@@ -5,16 +5,6 @@
   "pipeline.referenceFastaFai": "tests/data/reference/reference.fasta.fai",
   "pipeline.referenceFastaDict": "tests/data/reference/reference.dict",
   "pipeline.strandedness": "None",
-  "pipeline.starIndex": [
```

Comment:
Would it be cleaner for the STAR indexing test to be a separate test? Or would that just increase maintenance without real gain?

Reply:
That would increase travis usage without real gain.